src/code/components/notifications.js

```diff
@@ -8,27 +8,34 @@ class Notifications extends React.Component {
 
   static propTypes = {
     location: PropTypes.object,
-    notifications: PropTypes.array
+    notifications: PropTypes.array,
+    onAdvanceNotifications: PropTypes.func.isRequired,
+    onCloseNotifications: PropTypes.func.isRequired
   }
 
   static defaultProps = {
     notifications: []
   }
 
   render() {
-    const { location } = this.props,
-          style = location ? { left: location.left, top: location.top, right: 'auto' } : null,
-          messages = this.props.notifications.map((message, i) =>
-                        <div key={i} className="notification">{ t(message) }</div>);
-
+    const speaker = this.props.notifications.length > 0 ? <div className="fv-character"></div> : null,
+          message = this.props.notifications.length > 0 
```

Comment:
Not a big deal, but what would you think of just returning early with `null` (or whatever the correct idiomatic React is) first if `this.props.notifications.length === 0`? It would simplify this (it's not clear on first glance that every piece of this view is conditional on `length > 0`) and mean that you don't need to render anything at all.